less/index.less

```diff
@@ -54,6 +54,110 @@ table {
 }
 
 html, body {
-    height: 100%;
+	height: 100%;
+	font-size: 62.5%;
+	
+	
+}
+body {
+	font-size: 1.6rem;
+	
+	margin: 0 auto;
+	max-width: 1200px;
+}
+
+@resumefontcolor:#6495ED;
```

Comment:
so I see this variable but I the readme wants you to have two and a mixin but I can't find the mixin anywhere. Make sure to come back to this and include these things. You did a good job demonstrating you understand how to create one of these things and how to call/use it in the project so I'm happy to see that, just do the rest that's required for MVP and you'll be good